zookeeper-server/src/main/java/org/apache/zookeeper/audit/ZKAuditProvider.java

```diff
@@ -68,7 +68,9 @@ public static boolean isAuditEnabled() {
 
     public static void log(String user, String operation, String znode, String acl,
                            String createMode, String session, String ip, Result result) {
-        auditLogger.logAuditEvent(createLogEvent(user, operation, znode, acl, createMode, session, ip, result));
+        if (isAuditEnabled()) {
+            auditLogger.logAuditEvent(createLogEvent(user, operation, znode, acl, createMode, session, ip, result));
+        }
```

Comment:
For this method, all the callers have a pre-check for the `isAuditEnabled`

Reply:
Right,my mistake